test/config/akamai-purger.json

```diff
@@ -15,7 +15,9 @@
         "grpc": {
           "address": ":9099",
           "clientNames": [
-            "ocsp-updater.boulder"
+            "ra.boulder",
+            "ra1.boulder",
```

Comment:
I think `ra1.boulder` and `ra2.boulder` aren't necessary. In other configs we just use the service name.